sections/conclusion.tex

```diff
@@ -1,2 +1,36 @@
 \chapter{Conclusion}\label{ch:conclusion}
-There are probably still some bugs in the theme. If you should find one, then please submit it on \url{https://github.com/jkjaer/aauLatexTemplates}.
+The code that we overtook from last year for the search engine was incomplete and did not work.
```

Comment:
Should probably clarify the 'did not work' part. WordCount seemed to work OK, but we deemed it to be incomplete in terms of API endpoints, database structure, and code? Or am I mistaken?